README.md

```diff
@@ -223,6 +223,7 @@ Action to deploy to CloudHub 1.0. See [deploy_cloudhub_1_0/action.yml](deploy_cl
           <properties>
             <mule.env>${MULE_ENVIRONMENT}</mule.env>
             <securedKey>${encryptionKey}</securedKey>
+            <newrelic.apiKey>${NEWRELIC_API_KEY}</newrelic.apiKey>
```

Comment:
do we need this @tednguyendev 🤔

Reply:
Turned out it's needed 🙈 I wrote about it [here](https://github.com/Jollibee-Foods-Corporation/jfc-global-gcash-exp-api/pull/3), the part `Deploy to CloudHub`

Reply:
Got it @tednguyendev - glad that it works now hehe